Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -937,7 +937,14 @@ extension ClusterSystem {
             return invocation
         }
 
-        return try await ask.value
+        let reply = try await ask.value
+        if let error = reply.thrownError {
+            throw error
+        }
+        guard let value = reply.value else {
+            fatalError("Invalid RemoteCallReply, expected either value or thrownError to be set.")
```

Comment:
Let's just throw here -- make some new RemoteCallError case for it; I would not want to allow our system to crash only if someone managed to send us a bad message like that 👍